client/src/test/runTest.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path';
+import { runTests } from 'vscode-test';
+
+async function main() {
+    try {
+        // The folder containing the Extension Manifest package.json
+        // Passed to `--extensionDevelopmentPath`
+        const extensionDevelopmentPath = path.resolve(__dirname, '../../../');
```

Comment:
Lolwut. From `../..` onwards, I expect a comment explaining roughly where the path ends up.

Reply:
I know there’s the line “The folder containing the Extension Manifest package.json” but with all this client / server stuff this is still confusing.